templates/homev2_wifi.tpl

```diff
@@ -166,6 +172,10 @@ void submitValues() {
     client.stop();
     delay(1000);
   }
+
+  // check the server time for the validation of the certificate
+  ArduinoBearSSL.onGetTime(getTime);
```

Comment:
Should be done at the end of the setup()

Reply:
resolved in 982ff8c